services/graphql/src/graphql/definitions/order.js

```diff
@@ -18,6 +18,7 @@ extend type Mutation {
 
   orderName(input: OrderNameMutationInput!): Order! @requiresAuth @setAndUpdate(modelName: "order", path: "name")
   orderAdvertiser(input: OrderAdvertiserMutationInput!): Order! @requiresAuth @setAndUpdate(modelName: "order", path: "advertiserId")
+  orderExternalLinks(input: ExternalLinksMutationInput): Order! @requiresAuth @setAndUpdate(modelName: "order", path: "externalLinks")
```

Comment:
Is there a reason to have links on both the order and the line item?

Reply:
It was requested that way